conformance/tests/httproute-cross-namespace.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tests
+
+import (
+	"net/url"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/types"
+
+	"sigs.k8s.io/gateway-api/apis/v1alpha2"
+	"sigs.k8s.io/gateway-api/conformance/utils/http"
+	"sigs.k8s.io/gateway-api/conformance/utils/kubernetes"
+	"sigs.k8s.io/gateway-api/conformance/utils/roundtripper"
+	"sigs.k8s.io/gateway-api/conformance/utils/suite"
+)
+
+func init() {
+	ConformanceTests = append(ConformanceTests, HTTPRouteCrossNamespace)
+}
+
+var HTTPRouteCrossNamespace = suite.ConformanceTest{
+	ShortName:   "HTTPRouteCrossNamespace",
+	Description: "A single HTTPRoute in the gateway-conformance-web-backend namespace attaching to Gateway in other namespace",
```

Comment:
If the description tells me that it is a negative or positive test, it is better - code and words must match up. Else, the reader has to read the code to figure out the purpose of the test.
```suggestion
	Description: "A single HTTPRoute in the gateway-conformance-web-backend namespace attaches to Gateway in other namespace",
```

Reply:
Good call, I've updated most test descriptions, hopefully they're clearer now.